app/models/answer.rb

```diff
@@ -0,0 +1,3 @@
+class Answer < ApplicationRecord
+  belongs_to :question
```

Comment:
Probably a good idea to add some constraints as well, such as no empty content, max length limits, etc.